### PR TITLE
DnsClient PowerShell reference update

### PR DESCRIPTION
**Pull request by TimShererWithAquent**
[1869909](https://dev.azure.com/mseng/TechnicalContent/_workitems/edit/1869909)

### CODE REVIEW

**Comment by TimShererWithAquent** [comment]
Per email to Alexandru, this is ready for final review.

File: docset/winserver2022-ps/dnsclient/Add-DnsClientDohServerAddress.md [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName, ByValue)
+Accept wildcard characters: False
+```
+
+### -ThrottleLimit
+Specifies the maximum number of concurrent operations that can be established to run the cmdlet.
+If this parameter is omitted or a value of `0` is entered, then Windows PowerShell® calculates an optimum throttle limit for the cmdlet based on the number of CIM cmdlets that are running on the computer.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/dnsclient/Add-DnsClientDohServerAddress.md:152]:
```suggestion
If this parameter is omitted or a value of `0` is entered, then Windows PowerShell calculates an optimum throttle limit for the cmdlet based on the number of CIM cmdlets that are running on the computer.
```
```suggestion
If this parameter is omitted or a value of `0` is entered, then Windows PowerShell® calculates an optimum throttle limit for the cmdlet based on the number of CIM cmdlets that are running on the computer.
```

File: docset/winserver2022-ps/dnsclient/Get-DnsClientDohServerAddress.md [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+Required: False
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName, ByValue)
+Accept wildcard characters: False
+```
+
+### -ThrottleLimit
+Specifies the maximum number of concurrent operations that can be established to run the cmdlet.
+If this parameter is omitted or a value of `0` is entered, then Windows PowerShell® calculates an optimum throttle limit for the cmdlet based on the number of CIM cmdlets that are running on the computer.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/dnsclient/Get-DnsClientDohServerAddress.md:102]:
```suggestion
If this parameter is omitted or a value of `0` is entered, then Windows PowerShell calculates an optimum throttle limit for the cmdlet based on the number of CIM cmdlets that are running on the computer.
```
```suggestion
If this parameter is omitted or a value of `0` is entered, then Windows PowerShell® calculates an optimum throttle limit for the cmdlet based on the number of CIM cmdlets that are running on the computer.
```

Comment (JohanFreelancer9) [inline docset/winserver2022-ps/dnsclient/Get-DnsClientDohServerAddress.md:102]:
```suggestion
If this parameter is omitted or a value of `0` is entered, then Windows PowerShell calculates an optimum throttle limit for the cmdlet based on the number of CIM cmdlets that are running on the computer.
```
```suggestion
If this parameter is omitted or a value of `0` is entered, then Windows PowerShell® calculates an optimum throttle limit for the cmdlet based on the number of CIM cmdlets that are running on the computer.
```

File: docset/winserver2022-ps/dnsclient/Set-DnsClientDohServerAddress.md [hunk 3]
```diff
@@ -0,0 +1,270 @@
...
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByPropertyName, ByValue)
+Accept wildcard characters: False
+```
+
+### -ThrottleLimit
+Specifies the maximum number of concurrent operations that can be established to run the cmdlet.
+If this parameter is omitted or a value of `0` is entered, then Windows PowerShell® calculates an optimum throttle limit for the cmdlet based on the number of CIM cmdlets that are running on the computer.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/dnsclient/Set-DnsClientDohServerAddress.md:195]:
```suggestion
If this parameter is omitted or a value of `0` is entered, then Windows PowerShell calculates an optimum throttle limit for the cmdlet based on the number of CIM cmdlets that are running on the computer.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @TimShererWithAquent, kindly commit the few minor suggested changes to remove the registered trademark symbol. Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by TimShererWithAquent** [comment]
@tiburd Editorial changes accepted. Ready for final review.

**Review by get-itips — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by dariomws — APPROVED** [review]

**Comment by TimShererWithAquent** [comment]
Hi @tiburd - It looks like this is ready to merge.

**Pull request merged**